### PR TITLE
Spanish Translation of Tutorials [ Translated: 4/15 ] 

### PR DESCRIPTION
**Pull request by hellogaray**
I'm hoping to translate some of the tutorials to Spanish in order of most recently updated. I saw someone else is translating some of them too so if someone wants to review it that would be greatly appreciated. I tried to use a standard Spanish as much as possible, but since the language tends to vary a lot between countries it is possible I might have missed something.

### Translated: [ 4/15 ]
- [x] [ChimpLineByLine.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/es/ChimpanceLineaporLinea.rst) 
- [x] [CameraIntro.rst ](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/es/IntroC%C3%A1mara.rst)
- [ ] [DisplayModes.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/DisplayModes.rst)
- [x] [ImportInit.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/es/ImportarInit.rst)
- [ ] [MakeGames.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/MakeGames.rst)
- [ ] [MoveIt.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/MoveIt.rst) 
- [ ] [newbieguide.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/newbieguide.rst)  
- [ ] [PygameIntro.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/PygameIntro.rst)   
- [ ] [SpriteIntro.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/SpriteIntro.rst) 
- [ ] [SurfarrayIntro.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/SurfarrayIntro.rst) 
- [x] [tom_games2.rst ](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/es/tom_juegos2.rst)
- [ ] [tom_games3.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/tom_games3.rst)  
- [ ] [tom_games4.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/tom_games4.rst)  
- [ ] [tom_games5.rst ](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/tom_games5.rst) 
- [ ] [tom_games6.rst](https://github.com/hellogaray/pygame/blob/main/docs/reST/tut/tom_games6.rst)

### CODE REVIEW

File: docs/reST/tut/es/IntroCámara.rst [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+
+.. rst-class:: docinfo
+
+:Autor: by Nirav Patel
+:Contacto: nrp@eclecti.cc
+
+Pygame 1.9 viene con soporte para cámaras interconectadas, lo que le permite capturar
+imágenes fijas, ver transmisiones en vivo y hacer algo simple de visión por computadora. Este
+tutorial cubrirá todos esos casos de uso, proveyendo ejemplos de código en los que tu puedes basar
+tu aplicación o juego. Te puedes referir a  :mod:`reference documentation <pygame.camera>` por el API completo.
```
Comment (novialriptide) [inline docs/reST/tut/es/IntroCámara.rst:21]: nitpick: there's an extra space here.

**Comment by Starbuck5** [comment]
Thanks for working on this!

Couple thoughts:
- The camera module intro is very old and the camera module has changed since then, so it would be good if someone did a modernization pass on the English version first before committing to a Spanish translation.
- It would be nice if we didn't have to duplicate all the images between the two language versions

**Comment by hellogaray** [comment]
> 

I'm planning on translating more, if you have any suggestions on which ones I should start with as to give people time to update anything that might benefit from it before translating it. I can also update this once the original has been translated. I was trying to link the images to the original, but for some reason it wouldn't work. I'm not sure what I was doing wrong. I'll try it again and updated it if I figure it out.

**Comment by Starbuck5** [comment]
I looked again and the camera module intro [English] is aging well, I didn't see anything big to edit.

**Comment by hellogaray** [comment]
Changing to draft while I translate more tutorials.

**Comment by hellogaray** [comment]
> Thanks for working on this!
> 
> Couple thoughts:
> 
> * The camera module intro is very old and the camera module has changed since then, so it would be good if someone did a modernization pass on the English version first before committing to a Spanish translation.
> * It would be nice if we didn't have to duplicate all the images between the two language versions

I was able to fix the images, there is no need for a duplicate anymore so all will only stay on the original folder. I also updated the tutorial that had already been translated to use the images from the original folder.

**Comment by illume** [comment]
> I saw someone else is translating some of them too

Hey hey!

It seems you already saw on the website that there's already someone working on translating the tutorials to Spanish. Unfortunately there's some duplication of effort already, and no one doing the translations saw your work until now. Sorry about that.

We're just learning how to do all this, so haven't really got a good workflow yet. It's a bit complicated to sync and coordinate all that work between multiple people, and it's a bit higher intensity coordination than the usual pull requests. Github is not super well suited for collaborative writing work.

All the best, and sorry again for not getting back to you sooner.

**Comment by hellogaray** [comment]
> > I saw someone else is translating some of them too
> 
> Hey hey!
> 
> It seems you already saw on the website that there's already someone working on translating the tutorials to Spanish. Unfortunately there's some duplication of effort already, and no one doing the translations saw your work until now. Sorry about that.
> 
> We're just learning how to do all this, so haven't really got a good workflow yet. It's a bit complicated to sync and coordinate all that work between multiple people, and it's a bit higher intensity coordination than the usual pull requests. Github is not super well suited for collaborative writing work.
> 
> All the best, and sorry again for not getting back to you sooner.

I only saw the one file translated here on GitHub, but it was from a few months back so I decided to keep doing it. I did see a tweet (about looking for a translator) soon after so I decided to slow down just in case, so no real harm done! Anyways, I'll close this but please do feel free to contact me if you all need help with the translation in the future. Thanks!

**Pull request closed without merging**